tests/unitary/test_modules.py

```diff
@@ -9,3 +9,5 @@ def test_throw():
     c = boa.load(FIXTURES / "module_contract.vy")
     with boa.reverts("Error with message"):
         c.fail()
+    with boa.reverts("error message for devs"):
```

Comment:
if we are going to use the same line for both errors, we should probably use the kwarg for `reverts()` so it's clear what we are testing for